torch/onnx/symbolic_opset10.py

```diff
@@ -322,39 +322,79 @@ def isfinite(g, input):
     return __not_(g, __or_(g, inf_node, nan_node))
 
 
+def quantize_per_tensor(g, input, scale, zero_point, dtype):
+    dtype = sym_help._get_const(dtype, "i", "dtype")
+    zero_point = g.op("Cast", zero_point, to_i=sym_help.scalar_type_to_onnx[dtype])
+    scale = g.op("Cast", scale, to_i=torch.onnx.TensorProtoDataType.FLOAT)
+    return sym_help._quantize_helper(g, input, scale, zero_point)
+
+
+def dequantize(g, input):
+    return sym_help._dequantize_helper(g, input)[0]
+
+
 # https://github.com/pytorch/pytorch/wiki/PyTorch-ONNX-exporter#quantized-model-export
 class Quantized:
     domain = "quantized"
 
     # DequantizeLinear was added in opset version 10.
     @staticmethod
-    def linear(g, input_original, weight, weight_scale, weight_zero_point, bias, op_scale, op_zero_point):
-        input_value, input_scale, input_zero_point = sym_help._unpack_tuple(input_original)
+    def linear(g, q_input, q_weight, bias, op_scale, op_zero_point):
         # From https://pytorch.org/docs/master/generated/torch.nn.quantized.functional.linear.html
```

Comment:
Not clear how this comment applies to the line below it now.  Maybe move this comment down one line?
Or clarify this comment.

Reply:
are you referring to the comment `# From https://pytorch.org/docs/master/generated/torch.nn.quantized.functional.linear.html`? Do you mean it's better to put it above `def linear`?

Reply:
Yes I'm referring to that comment.

I interpreted the comment as providing the source of truth for the later comments that assert the types of the input arguments (e.g. `# input (Tensor) – Quantized input of type torch.quint8`).
I guess either:
* just remove the comment, or
* move it down to above line 345 so it's clear that it is providing context for the next comment (`# weight (Tensor) – Quantized weight of type torch.qint8`)

Reply:
removed. Verified it is not needed to manually pass the dtype.